.eslintrc.js

```diff
@@ -129,7 +129,12 @@ module.exports = {
       },
     },
     {
-      files: ['ui/**/*.test.js', 'ui/__mocks__/*.js', 'shared/**/*.test.js'],
+      files: [
+        'ui/**/*.test.js',
+        'ui/__mocks__/*.js',
+        'shared/**/*.test.js',
+        'test/jest/*.js',
```

Comment:
❤️

Reply:
Though, per this [conversation](https://github.com/MetaMask/controllers/pull/579#discussion_r700590783) I had yesterday with @rekmarks this probably merits some conversations in the [eslint config repo](https://github.com/MetaMask/eslint-config)

Reply:
Ah! I didn't know that existed. Should I open a PR there? OTOH the `test/jest` directory seems pretty specific to this repo.

Reply:
Maybe raise the conversation there? I think that's probably the protocol here.

Reply:
Ended up removing the matcher, so this change isn't needed.